Team9Project/src/com/indragie/cmput301as1/ExpenseClaimListActivity.java

```diff
@@ -45,15 +43,23 @@
 	
 	private static final int ADD_EXPENSE_CLAIM_REQUEST = 1;
 	private static final int EDIT_EXPENSE_CLAIM_REQUEST = 2;
+	
 	private static final String EXPENSE_CLAIM_FILENAME = "claims";
 
 	//================================================================================
 	// Properties
 	//================================================================================
 	
-	private ArrayList<ExpenseClaim> claims;
-	private ExpenseClaimArrayAdapter adapter;
+	private static ArrayList<ExpenseClaim> claims;
+	private static ExpenseClaimArrayAdapter adapter;
 	private int longPressedItemIndex;
+	
+	public static ExpenseClaimArrayAdapter getAdapter () {
```

Comment:
Adapter and claims list shouldn't be exposed publicly (related to the comment above). The expense claim array should be passed inside the `Intent` (as an extra) to `ExpenseClaimSortActivity`, and then the sorted array should be retrieved from the result intent (inside `onActivityResult()`) once the sorting activity finishes.